src/index.html

```diff
@@ -15,8 +15,106 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
```

Comment:
The `<link>` tag starting on line 18 should be indented with 2 spaces relative to the `<head>` tag.